packages/documentation/src/stories/foundation/palette/palette.docs.mdx

```diff
@@ -0,0 +1,27 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import * as paletteStories from './palette.stories';
+
+<Meta of={paletteStories} />
+
+<div className="docs-title">
+  # Palettes
+</div>
+
+<div className="lead">
+  Easily apply colors to a section of your page using predefined color sets.
+</div>
+
+There are four different palettes:
+- **Default** - The standard palette used when no other palette is specified.
+- **Alternate** - Used to differentiate alternating sections from the body, without strong emphasis or highlighting.
+- **Accent** - A complementary color used for highlights and emphasis.
+- **Brand** - The primary color for the brand.
+
+Palettes may also include a specific text color that is different from the body color.
+You can apply this text color to any element by using the `.palette-text` class.
+
```

Comment:
I'm missing documentation for people who'd like to implement palettes, without using an entry file. What files are necessary to just get palettes to work and where can they be found?

Reply:
You're right, I created a separate ticket for that since it requires changes to the StylesPackageImport component: https://github.com/swisspost/design-system/issues/4025